api/api.go

```
@@ -16,6 +16,7 @@ type ResourceAPI struct {
 }
 
 var ErrNumArgs = errors.New("wrong number of args")
+var Marshal = json.Marshal
```

Lets not introduce a global variable to increase coverage. If this error can never occur we can just ignore the error in the code path.